monitor/census.go

```diff
@@ -2073,6 +2082,17 @@ func AILiveVideoAttempt() {
 	stats.Record(census.ctx, census.mAILiveAttempts.M(1))
 }
 
+func AINumOrchestrators(count int, modelName string) {
+	if !Enabled {
+		return
+	}
```

Comment:
Checking `if monitor.Enabled` inside, in here and not like [in other cases](https://github.com/search?q=repo%3Alivepeer%2Fgo-livepeer+monitor.Enabled&type=code)